pkg/controllers/dnspolicy/dns_helper_test.go

```diff
@@ -1,13 +1,15 @@
-//go:build unit
-
-package dns_test
+package dnspolicy
 
 import (
 	"context"
 	"reflect"
 	"sort"
 	"testing"
 
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/dns"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/traffic"
+	testutil "github.com/Kuadrant/multicluster-gateway-controller/test/util"
```

Comment:
nit: imports moved about and not grouped 

There's some openshift tool somewhere that can be used to group imports as desired i believe. Should look into that and run it on CI.